### ISSUE TITLE
Why so may appleID sign-ins?

### ISSUE THREAD

**Issue by Steph1116** [gcobb321/icloud3_v3#223]
I've noticed a higher amount of emails lately saying my Apple ID was used to sign in. For example I received 7 emails Friday, 8 on Saturday and 7 on Sunday. I have set up icloudv3 with the default settings and I'm running v3.0.rc7.1.1. I literally installed it like I was supposed to, put my Apple info and sent the pairing code they sent me and added my devices. So I'm not sure what's up. Thanks

**Comment by gcobb321** [comment]
Apple will send an email every time there is a login with a password. iCloud3 receives a security token when that happens that can then be used to validate account access so a password is not needed. The token is saved as a session cookie and is presented about every 30-minutes when iCloud3 asks for Location data. It expires after a short time, which is determined by Apple,  and the password is requested again. The password is sent and the cycle starts again. You receive an email when that happens. This is all determined by Apple and iCloud3 has no control over it. 

About the only thing you can do is to create an email filter that moves those emails into an email folder, bypass the inbox and Mark them as read.

**Comment by Steph1116** [comment]
Which is probably the worst idea because if your account gets hacked then you may need that email notification to tell you something logged into your account. But thanks for closing the issue without discussion. I brought it up here because I went from like one to two per day to 8. I think when I first installed it I went a few days without an email. I guess as a plain old user and not the brains behind the creation, I'm just an idiot and I'm probably doing something wrong.

**Comment by gcobb321** [comment]
I had been displaying when an authentication was being done that indicates if it was a token or token +password and they were displayed in the _Event Log_ about every 1/2 hour so I moved the message to the _Event Log > Side Tracking Monitors_. The emails have also been an issue forever with other programs that remain permanently logged into the Apple account.

**Comment by JtwoA** [comment]
> Which is probably the worst idea because if your account gets hacked then you may need that email notification to tell you something logged into your account. But thanks for closing the issue without discussion. I brought it up here because I went from like one to two per day to 8. I think when I first installed it I went a few days without an email. I guess as a plain old user and not the brains behind the creation, I'm just an idiot and I'm probably doing something wrong.

This is probably the most juvenile and disrespectful reply I could imagine to this.  Gary isn't paid for his work, he's doing this because he's giving back to the HA community.  

This question of yours has been asked and answered previously.  Gary is held to working within the confines of the Apple world.  They don't consult with him on potentially breaking decisions or how they can best write things that will help him work within their world.

Seems to me this integration doesn't meet your needs and you'd be better off writing your own.

**Comment by Steph1116** [comment]
1.Turning off one of the line of defence protecting  the Apple account from hackers or what not is a dumb idea. Those emails are there to inform the person something is happening. With this creation as complex as it is and created by highly intelligent people, I just naturally assumed they would know this.
2. I assumed this was an issue/discuss location, not shut down by god if they find their answer to be the solve all be all. 
3. I don't know why you people always insist that if the question has been asked in the past that it's always solved. Those past questions were related to the same topic but they occurred on different firmware versions of icloud3. They could be something different that user did or didn't do. There's too many factors to list. So there's nothing wrong with asking the same question. Also it's more recent and at the top, so other people on the same version, having the same problem can reference it.
4. I posted the issue, he replied and played god and assumed his answer was it and closed the ticket. I was not finished discussing and going back and forth for other solutions or other user feedback back. If it then gets solved, we can agree and close the topic. No one will burn up or an asteroid will hit the earth if a topic stays open for a while, so why close it so fast. This is the users only way to contact the developers of the add-on/integration. 
5. And lastly, as the user, I am no where near smart enough to make such a great integration. And for the developer to just outright say, yeah we'll do that, closed the issue, it was like telling me to shut up. It's disrespectful to not allow the issue to be discussed, even though it's been discussed before in older versions. Give your users the chance to understand issues and the complexity of the inner working. Don't shut them up because you feel it's closed. Not everyone is as intelligent as you and need a bit more explaining and discussion. By creating this for free and doing follow up for free, doesn't mean your right or wrong. By me being a user same goes for me. But every single user should still have a chance to post an issue and discuss it regardless of past posts. 

Thanks

**Comment by JtwoA** [comment]
> 1.Turning off one of the line of defence protecting the Apple account from hackers or what not is a dumb idea. Those emails are there to inform the person something is happening. With this creation as complex as it is and created by highly intelligent people, I just naturally assumed they would know this. 2. I assumed this was an issue/discuss location, not shut down by god if they find their answer to be the solve all be all. 3. I don't know why you people always insist that if the question has been asked in the past that it's always solved. Those past questions were related to the same topic but they occurred on different firmware versions of icloud3. They could be something different that user did or didn't do. There's too many factors to list. So there's nothing wrong with asking the same question. Also it's more recent and at the top, so other people on the same version, having the same problem can reference it. 4. I posted the issue, he replied and played god and assumed his answer was it and closed the ticket. I was not finished discussing and going back and forth for other solutions or other user feedback back. If it then gets solved, we can agree and close the topic. No one will burn up or an asteroid will hit the earth if a topic stays open for a while, so why close it so fast. This is the users only way to contact the developers of the add-on/integration. 5. And lastly, as the user, I am no where near smart enough to make such a great integration. And for the developer to just outright say, yeah we'll do that, closed the issue, it was like telling me to shut up. It's disrespectful to not allow the issue to be discussed, even though it's been discussed before in older versions. Give your users the chance to understand issues and the complexity of the inner working. Don't shut them up because you feel it's closed. Not everyone is as intelligent as you and need a bit more explaining and discussion. By creating this for free and doing follow up for free, doesn't mean your right or wrong. By me being a user same goes for me. But every single user should still have a chance to post an issue and discuss it regardless of past posts.
> 
> Thanks

This isn't the appropriate place for debate.  If you'd like to reach out privately, feel free.  I stand by my comment.

Back to our regularly scheduled Issue #233

**Comment by Steph1116** [comment]
Yes god, sorry god

**Comment by Steph1116** [comment]
Yes sir 🫡

**Comment by Steph1116** [comment]
I'm glad this integration has bouncers, keep the user in line.

**Comment by Steph1116** [comment]
https://github.com/gcobb321/icloud3_v3/issues/72 this was raised 8 months ago, a lot could have changed since then and the answers may not be relevant today. It was discussed and then closed after a period of time. It says pretty much close to what I posted, but different version#. I can't find more, though there could be. I reacted  because I didn't get a fighting chance to discuss my problem, let alone just reply before it got closed. But now bouncer boy jumps in and the issue gets buried and turned into something else. Don't have issues activated if you don't want to discuss the users issues. I never once called out the creators or installed them or their code, I commented on the "quick shut up and deal with it". And I still have the issue of a million emails. But I don't want to throw out the integration because it's really great and helps a lot in my use case. Anyhow, I'll know for the future that this isn't the place to come for help. 👍

**Comment by JtwoA** [comment]
@gcobb321 Apologies for this thread going so FAR off the rails.  I've blocked this user as he only seems interested in arguing just to argue in the public eye vs a sidebar conversation.

**Comment by gcobb321** [comment]
I only closed the issue because it has been asked and answered many many times over the years and I really had nothing more to add. No disrespect has intended by closing your issue. Feel free to reopen it if you want but I have nothing more I can add as to that can be done to minimize the emails.  Apple controls everything. I control nothing. I can not change the code any more than I have and we all must figure out a way to live in the Apple-verse. My solution is what I do.

 The current download count of iCloud3 v2 is over 11,300 the last time I looked at it. I’m sure everyone is concerned with security and has found a way to live with and deal with the emails. If you find a way that works for you that is different, please share and I’ll put it in the User Guide.

**Comment by Steph1116** [comment]
Right, and by me posting about it, I wanted to possibly get advice from other users on how they manage it. Still remember from a user perspective, this is the first time I'M having the issue that was getting crazy, so I reached out to only be shut down less then a few hours with one suggestion, being that I close a line of defence. I've posted on a bunch of issue boards and almost always there was a back and forth for each user with other users and staff to find solutions or to figure out it couldn't be fixed. Nothing against you or your code or anything, but it's the first time that I actually got shut up from any help board. I'm floored. And the best answer, well we headed it a million times so go figure it out yourself. Again, I'm new to this integration and the current version on this topic, that I could find, hasn't been brought up. Also it was working without much emails and then an explosion after the last few updates. So I figured maybe there's an issue in the codes end. I don't know, I use it not write it, hence bring my issue to "issues". You see the issues everyday, I don't have updates from the 11,000 or so users to update me or bring an issue to me. Anyhow, I'll leave this post up for a bit and then delete the whole thing and not bother you people again.
